### PR TITLE
Cross-Platform Juggle with Flutter

### PR DESCRIPTION
**Pull request by denzilferreira**
My experiences with Flutter and introduction.

### CODE REVIEW

**Review by jarnovayrynen — APPROVED** [review]

File: _posts/2020-12-11-cross-platform-juggle-with-flutter.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+author: denzilferreira
+excerpt: >
+  Creating a mobile app is usually tasked to two distinct teams, working in tandem to produce the same experience for Android & iOS users. What if I told you that there is a platform in town that offers a shortcut to create a mobile app in record time, simultaneously on Android, iOS, and maybe beyond towards a web, and as a desktop app?
+tags:
+ - Software Development
+ - Dart
+ - Flutter
+---
+
+> Author disclaimer: This is my personal view on Flutter and may not reflect the opinion of other Solitans nor the company's stance on Flutter.
```
Comment (jarnovayrynen) [inline _posts/2020-12-11-cross-platform-juggle-with-flutter.md:13]: I think you don't necessarily need this disclaimer - but it doesn't too much harm either.

**Comment by jarnovayrynen** [comment]
Let's not merge before Denzil gives an OK with his final testings on Docker...

**Comment by denzilferreira** [comment]
Tested post on Docker, images OK now.

File: _posts/2020-12-11-cross-platform-juggle-with-flutter.md (hunk 1 above)
Comment (denzilferreira) [reply _posts/2020-12-11-cross-platform-juggle-with-flutter.md:13]: I can remove if you don't think it's needed :)

**Pull request merged**